### ISSUE TITLE
@stacksjs/commerce

### ISSUE THREAD

**Issue by chrisbbreuer** [stacksjs/stacks#268]
### Clear and concise description of the problem

Being able to easily create a complex online shop, or simple transactions/subscriptions.

### Suggested solution

- [ ] Coupons
- [ ] Subscriptions
- [ ] Gift Cards
- [ ] One-off purchases
- [ ] [Algolia integration](https://www.algolia.com/blog/ecommerce/) & other search engines
- [ ] Prebuilt UI components & generated API endpoints
- [ ] Printer Integration
- [ ] QR & Bar Code Integration
- ...

In other words, translate the Ordern codebase.

### Validations

- [X] Follow our [Code of Conduct](https://github.com/stacksjs/.github/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guide](https://github.com/stacksjs/contributing).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

**Comment by glennmichael123** [comment]
Shipped — `storage/framework/core/commerce/` is the active module today (`@stacksjs/commerce`, ~17231 LOC of TypeScript source). Closing this epic-level tracker; file targeted issues for any specific gap in the current implementation.
